File: toast/src/main/java/com/toast/ToastParam.java
package com.toast;

import android.content.Context;
import android.widget.Toast;

/**
 * Created by amit on 24/1/17.
 */

public class ToastParam {

    /**
     * The Message.
     */
    String message = "";
    /**
     * The Context.
     */
    Context context;
    /**
     * The Message res id.
     */
    int messageResId = -1;

    /*Typeface for the specific alert*/
    String typeface;

    /**
     * The Duration.
     */
    int duration = Toast.LENGTH_SHORT;

}

File: toast/src/main/java/com/toast/ToastParam.kt
package com.toast

import android.content.Context
import android.widget.Toast

/**
 * Created by amit on 24/1/17.
 */

class ToastParam {

    /**
     * The Message.
     */
    internal var message = ""
    /**
     * The Context.
     */
    internal var context: Context? = null
    /**
     * The Message res id.
     */
    internal var messageResId = -1

    /*Typeface for the specific alert*/
    internal var typeface: String? = null

    /**
     * The Duration.
     */
    internal var duration = Toast.LENGTH_SHORT

}
